docs/user-guide/time-and-costs/progress-tracking/README.md

```diff
@@ -2,93 +2,111 @@
 
-As an alternative to the manual progress tracking mode above, you can configure your OpenProject system to associate work packages statuses with fixed values for % Complete .
-In the Administration, please navigate to Work packages -> Settings and change “Calculate the work package done ratio” to
-“Use the work package status”.
+In Status-based progress reporting mode, Work is not a required value. However, if Work is set, Remaining work is automatically calculated. To make this link clear and transparent, clicking on any of the three values to modify them will display the following pop-over:
 
-![Change calculate the work package done ratio](image-20221102110738283.png)
+**[IMG: Status-based progress reporting]**
 
-In this mode, OpenProject does _not_ allow you to manually modify the
-**% Complete (earlier called Progress (%))** field in the work packages.
-Instead, the **% Complete** value will be set automatically based on the work package
-status configuration page.
+This allows you to edit %&nbsp;Complete (by changing status) or Work and get a preview of the updated Remaining work before saving changes.
 
-Within the Administration, navigate to Work package -> Status and select the status you want to edit regarding the progress tracking. Here you can select the desired progress percentage from the drop-down menu in the progress line and add it to the status. Please do not forget to save your changes.
+>**Note:** In the upcoming version, statuses cannot have an empty %&nbsp;Complete values in status-based progress reporting mode.When upgrading, all statuses that do not have a value will take the default value of 0%.
+
+
+## Hierarchy totals
+
+OpenProject will automatically show totals for Work, Remaining work and % Complete in a work package hierarchy (any parent with children). These appears in a work package table as a number with a Σ sign next to it, indicating that it is a sum of the values of the parent _and_ children.
+
+**[IMG: Hierarchy example]**
+
+> **Note**: The total %&nbsp;Complete value of a hierarchy is a weighted average tied to Work. For example, a feature with Work set to 50h that is 30% done will advance the sum of %&nbsp;Complete of the parent more than a feature with Work set to 5h that is 70% done. 
+
+
+### Excluding certain work packages from totals
+
+In some cases, you might want to exclude certain work packages (like those with status *rejected*) from sum calculations of the parent. Administrators can define these exclusions by going to the [Administration settings for any status] and check a new option called "Exclude from calculation of totals in hierarchy". All work packages with this status will then be excluded when calculating the sum value for the parent (for all fields: Work, Remaining work and %&nbsp;Complete).
+
+A small info icon will appear next to excluded values to remind you of this fact:
+
+**[IMG: Excluded work packages warning]**
```

Comment:
Excluding statuses is not available in 14.0

Reply:
And we should talk about "total calculations" rather than "sum calculations" as total is the term we use in our app.

Reply:
Replaced the words "sum" with "total", thanks.

Reply:
And commented out the excluded statuses bit for now.